docker-compose.test.yml

```diff
@@ -0,0 +1,59 @@
+version: "2.1"
+services:
+  postgres:
+    image: postgres:14-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: docker
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+    ports:
+      - "5432"
+      - "5431:5432"
+    networks:
+      - internal
+  redis:
+    image: redis:alpine
+    restart: always
+    ports:
+      - "6379"
+      - "6378:6379"
+    networks:
+      - internal
+  loki: 
```

Comment:
Loki isn't necessary for any pinejs stuff is it?

Reply:
correct, sorry for copy paste from open-balena test dc.yml